### PR TITLE
Firewall:NAT:Port Forward - missing (disabled Linked rule) - fix

### PR DESCRIPTION
**Pull request by opnsenseuser**
@fichtner -> the "disabled Linked rule" symbol was missing in the current port forwarding settings
see also https://github.com/opnsense/core/issues/2968

### CODE REVIEW

**Comment by fichtner** [comment]
no need for "@" mentions in commits :)

File: src/www/firewall_nat.php [hunk 1]
```diff
@@ -516,6 +516,10 @@ function delete_id($id, &$array)
                       <td><i class="fa fa-arrows-h fa-fw text-success"></i></td>
                       <td colspan="12"><?=gettext("Linked rule");?></td>
                     </tr>
+                    <tr>
+                      <td><i class="fa fa-arrows-h fa-fw text-muted"></i></td>
+                      <td colspan="12"><?=gettext("Disabled Linked rule");?></td>
```
Comment (fichtner) [inline src/www/firewall_nat.php]:
```suggestion
                      <td colspan="12"><?=gettext("Disabled linked rule");?></td>
```

**Comment by fichtner** [comment]
So this is nice, but I have a request: Firewall: Rules have a nicer way of dealing with this. Would you mind adapting this page to use the same approach?

screenshot 2018-11-25 at 12 52 37 pm

**Comment by opnsenseuser** [comment]
@fichtner yes!

**Comment by fichtner** [comment]
Merged manually via https://github.com/opnsense/core/commit/8ea6b72a0 -- superb, thanks!

**Pull request closed without merging**